docs/VR/AddCommand/index.md

```diff
@@ -11,12 +11,23 @@ Purpose
 
 ### Request
 
-If the application sends `AddCommand` with the `vrCommands` parameter then SDL must maintain a list of the added `vrCommands`.  
+If the application sends `AddCommand` with the `vrCommands` parameter then SDL Core will maintain a list of the added `vrCommands`.  
 For each AddCommand, only the first item in the `vrCommands` array shall be added to the list.
 
-Whenever the internal list of added `vrCommands` is updated SDL must:  
-* construct the `vrHelp` and `helpPrompt`  parameters using the data from the list SDL created internally  
-* send these parameters to the HMI via the `SetGlobalProperties` RPC
+Whenever the internal list of added `vrCommands` is updated SDL Core must:  
+* Construct the `vrHelp` and `helpPrompt`  parameters using the data from the list SDL created internally.  
+* Send these parameters to the HMI via the `SetGlobalProperties` RPC.
+
+If the application sends a `CreateInteractionChoiceSet` RPC request that contains `vrCommand` parameters, SDL Core will send the HMI a `VR.AddCommand` request.
```

Comment:
This part suggests plural params:
```contains `vrCommand` parameters```

This part suggests a single RPC:
```a `VR.AddCommand` request```

Can we modify the second predicate to suggest it will send at least one RPC, one RPC per choice, or similar?